lib/pacemaker/pcmk_output.c

```diff
@@ -1864,6 +1864,65 @@ attribute_xml(pcmk__output_t *out, va_list args)
     return pcmk_rc_ok;
 }
 
+PCMK__OUTPUT_ARGS("rule-check", "const char *", "int", "const char *")
+static int
+rule_check_default(pcmk__output_t *out, va_list args)
+{
+    const char *rule_id = va_arg(args, const char *);
+    int result = va_arg(args, int);
+    const char *error = va_arg(args, const char *);
+
+    switch (result) {
+        case pcmk_rc_within_range:
+            return out->info(out, "Rule %s is still in effect", rule_id);
+        case pcmk_rc_ok:
+            return out->info(out, "Rule %s satisfies conditions", rule_id);
```

Comment:
I don't think this is very intuitive... for text output, I wonder if should we combine both `pcmk_rc_within_range` and `pcmk_rc_ok` into one case for "Rule %s is in effect". Or something like `"Rule %s is in effect: within date range"` and `"Rule %s is in effect: satisfies cron conditions"`

Reply:
I wouldn't bother at this point